test/test_helper.rb

```diff
@@ -100,3 +100,10 @@ class TestCase
     include Rails::Controller::Testing::Integration
   end
 end
+
+# ensure any mocha version is correctly required
+begin
+  require 'mocha/minitest'
```

Comment:
why this?

Reply:
https://github.com/freerange/mocha/issues/324
i happened to personnaly ran into this...
could be removed obvs

Reply:
actually this is a little bothering me.. i think it could be quite bothering for the gem's users if something goes really wrong cause i'm not doing anything with the error..
i'll remove that, and open an issue / add to the docs so it's referenced somewhere

Reply:
I think if you remove this, it won't break travis. So remove it if you can

Reply:
yes i'll open an issue for referencing